src/main/java/bit/javaoop/Employee.java

```diff
@@ -0,0 +1,31 @@
+package bit.javaoop;
+
+public class Employee {
+    private double salary;
+    private String name;
+
+    public Employee(int salary, String name) {
+        this.salary = salary;
+        this.name = name;
+    }
+
+    public boolean isSatisfied() {
+        return this.salary > 10_000;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public void setSalary(double salary) {
```

Comment:
Niepotrzebne i nieużywane settery i gettery, generalnie z tymi setterami i getterami to jest tak, że powinno się je tworzyć wyłącznie wtedy kiedy potrzebujemy ich użyć albo takie jest przeznaczenie naszej klasy, że np. powinna być możliwość zmieniania pensji pracownika wtedy zostawiamy setter. Ale nie generujemy ich tak od kopa bo wtedy tworzy nam się jedynie niepotrzebny boilerplate